docs/src/operations/requirements.md

```diff
@@ -55,21 +55,24 @@ other supported target platforms (MacOS and Windows) today and suggested for lin
 be required there as well
 
 ## Networking
-Internet service should be at least 1GBbit/s symmetric, commercial. 10GBit/s preferred (especially for mainnet-beta).
-A dedicated public IP address is preferred.
+A commercial connection with a public IPv4 address is required.
```

Comment:
What does "commercial" mean here? I think we should either drop this language or spell out the requirements